agent-inject/agent/config.go

```diff
@@ -104,7 +104,7 @@ func (a *Agent) newConfig(init bool) ([]byte, error) {
 		},
 		AutoAuth: &AutoAuth{
 			Method: &Method{
-				Type: "kubernetes",
+				Type:      "kubernetes",
```

Comment:
Random whitespace? Can you run `gofmt` on this?

Reply:
Done. It also fixed the formatting on other files.